ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/results-visual/result-item.collection.tsx

```diff
@@ -274,6 +283,41 @@ const ResultCards = ({ mode, setMode, selectionInterface }: Props) => {
                         measure={measure}
                         index={index}
                         width={width}
+                        draggable={true}
+                        onDragStart={(event: React.DragEvent) => {
+                          const dragPayload: any = {
+                            action: 'add',
+                          }
+
+                          const selectedResults = selectedResultsRef.current
+
+                          if (Object.entries(selectedResults).length > 0) {
+                            dragPayload['items'] = Object.values(
+                              selectedResults
+                            ).map((result) => {
+                              return {
+                                id: result.plain.id,
+                                metacardType: '',
```

Comment:
❓ how come keep metacardType blank?

Reply:
I think this can be removed since the metacard type is set on the backend.

Reply:
Removed.